pkg/api/types.go

```diff
@@ -2450,3 +2450,70 @@ const (
 	// "default-scheduler" is the name of default scheduler.
 	DefaultSchedulerName = "default-scheduler"
 )
+
+// Address of a cluster
```

Comment:
That was because we were still sorting out the API groups machinery. Adding
directly to a new group is the way to go now.

On Mon, Feb 15, 2016 at 3:23 PM, Chao Xu <notifications@github.com> wrote:

> In pkg/api/types.go
> <https://github.com/kubernetes/kubernetes/pull/21190#discussion_r52953017>
> :
>
> > @@ -2450,3 +2450,70 @@ const (
> >  	// "default-scheduler" is the name of default scheduler.
> >  	DefaultSchedulerName = "default-scheduler"
> >  )
> > +
> > +// Address of a cluster
>
> Actually according to the doc here
> <https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api_changes.md#so-you-want-to-change-the-api>
> and here
> <https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api_changes.md#incompatible-api-changes>,
> we should add new API to the extensions group first, and then graduate it
> to its own group.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/21190/files#r52953017>.
>